File: ScaleformUI_Lua/src/PauseMenu/Items/BasicTabItem.lua
BasicTabItem = setmetatable({}, BasicTabItem)
BasicTabItem.__index = BasicTabItem
BasicTabItem.__call = function()
    return "BasicTabItem", "BasicTabItem"
end

---@class BasicTabItem
---@field Label string
---@field Parent BasicTabItem

---Creates a new BasicTabItem.
---@param label string
---@return BasicTabItem
function BasicTabItem.New(label, labelFont)
    local data = {
        Label = label or "",
        LabelFont = labelFont or ScaleformFonts.CHALET_LONDON_NINETEENSIXTY,
        Parent = nil
    }
    return setmetatable(data, BasicTabItem)
end

File: ScaleformUI_Csharp/Menus/Pause Menus/PauseMenu/Items/BasicTabItem.cs
using ScaleformUI.Elements;

namespace ScaleformUI.PauseMenu
{
    public class PauseMenuItem
    {
        public ItemFont LabelFont = ScaleformFonts.CHALET_LONDON_NINETEENSIXTY;
        public ScaleformLabel Label { get; set; }
        public TabLeftItem Parent { get; set; }
        public PauseMenuItem(string label)
        {
            Label = label;
        }
        public PauseMenuItem(string label, ItemFont labelFont)
        {
            Label = label;
            LabelFont = labelFont;
        }
    }
}
